client/src/components/Auth/SignUp.jsx

```diff
@@ -8,20 +8,18 @@ import styled from "styled-components";
 
 import axios from "axios";
 
-
-import Modal, { ModalProvider, BaseModalBackground } from "styled-react-modal";
-
-
-const StyledModal = Modal.styled`
-  width: 20rem;
-  height: 20rem;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  background-color: white;
-  opacity: ${props => props.opacity};
-  transition: opacity ease 500ms;
-`;
+import Modal from "styled-react-modal";
+
+// const StyledModal = Modal.styled`
+//   width: 20rem;
+//   height: 20rem;
+//   display: flex;
+//   align-items: center;
+//   justify-content: center;
+//   background-color: white;
+//   opacity: ${props => props.opacity};
+//   transition: opacity ease 500ms;
+// `;
 
 const FormContainer = styled.div`
 display: flex;
```

Comment:
this looks like it would have been better served as an inline style since it's just one line unless you plan to do more with it than just this?

Reply:
I believe I had some ideas for this one, but I think I'll take your suggestion on this and make it inline for now. I'll have it implemented on the next PR.

Reply:
Wait, I'm not sure what you're talking about with regards to 1 line

This is what I see 
```const FormContainer = styled.div`
display: flex;
flex-direction: column;
justify-content: center;
align-items: center;
text-align: center;
width: 100%
height: 100%;
border-radius: 25px;
background:white;
border: 3px solid gold;
`;```